src/RA_httpthread.cpp

```diff
@@ -388,6 +388,8 @@ BOOL RAWeb::DoBlockingHttpGet(const std::string& sRequestedPage, std::string& Re
 
                     while (nBytesToRead > 0)
                     {
+
+                        // TODO: Replace pData's type with DataStream after PR #23 has been merged
```

Comment:
PR 23 has been merged. please address or remove this comment.

Reply:
Sure.